### PR TITLE
UI enhancements

### LINKED ISSUE #330: More spinners
When you see this:
updater-confusion

You ask:
- is it doing anything ?
- do I need to input something or click the button at the bottom ?

Suggestions:
- [ ] hide/disable the bottom part while the upgrade is running
- [ ] add spinners behind each step while they are running

The only hint I had that it was still working was that my laptop fan was loud and `top` showed activity.

@jancborchardt @VicDeo

### PR DESCRIPTION
**Pull request by VicDeo**
Show spinners for all steps Fixes #330 
Links top logo in updater to ownCloud, adds autoredirect to ownCloud after the successful update. Closes https://github.com/owncloud/updater/issues/344

### CODE REVIEW

**Comment by VicDeo** [comment]
@PVince81

File: pub/js/main.js [hunk 1]
```diff
@@ -179,25 +183,32 @@ $(function () {
...
 							? $.post($('#meta-information').data('endpoint'), {command: 'upgrade:restartWebServer'})
 							: $.Deferred()
 							;
 				})
 				.then(function (response) {
-					handleResponse(response, function () {}, '#step-finalize');
+					handleResponse(response, function () {accordion.showContent('#step-finalize');}, '#step-finalize');
 					return response.error_code === 0
-							? $.post($('#meta-information').data('endpoint'), {command: 'upgrade:postUpgradeCleanup'})
+							?  accordion.setCurrent('#step-finalize')
```
Comment (PVince81) [inline pub/js/main.js]: be careful with the `?` operator, priorities are often not intuitive like we think. I recommend enforcing priorities with parenthesis.

**Comment by VicDeo** [comment]
@PVince81 updated

File: pub/js/main.js [hunk 2]
```diff
@@ -179,25 +195,38 @@ $(function () {
...
 							: $.Deferred()
-							;
+					);
 				})
 				.then(function (response) {
 					handleResponse(response, function () {}, '#step-finalize');
 					if (response.error_code === 0){
 						accordion.setDone('#step-done');
-						accordion.setContent('#step-done', 'All done!');
+						accordion.setContent('#step-done', 'All done! Redirecting you to your ownCloud.');
```
Comment (PVince81) [inline pub/js/main.js]: use "product name" here if possible due to theming ? probably irrelevant for the updater anyway...

**Comment by PVince81** [comment]
Patched 9.1.2RC2 and updated to daily, spinners look nice :+1:

**Pull request merged**